### PR TITLE
Add shecoded page

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Adds fab page designed by @aspittel and @lianafelt

### CODE REVIEW

File: app/views/pages/shecoded.html.erb [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  <body>
+    <div class="content">
+      <h1>Nevertheless, She Coded</h1>
+      <img src="<%= asset_path "SheCodedLogo.png" %>" alt="shecoded logo" />
+      <h2 class="mobile-only">👩🏿‍💻👩‍💻👩🏾‍💻</h2>
+      <div class="text">
+        <b>International Women's Day is next week and we want to celebrate the
+          heck out of it.</b>
+        <p>
+          Since Balance for Better is the official 2018 theme for IWD, we're
```
Comment (Glennmen) [inline app/views/pages/shecoded.html.erb]: Shouldn't this be `2019`?

File: app/views/pages/shecoded.html.erb [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+          excited to celebrate women and non-binary people's bragging rights in
+          software development and gather advice for allies to take action. For
+          those of you who participated last year, we encourage you to submit
+          another post. And if you're new to the celebration, we welcome you to
+          contribute to the collective.
+        </p>
+        <p>
+          On Thursday March 8th, we'll be boosting your stories all over dev.to
```
Comment (Glennmen) [inline app/views/pages/shecoded.html.erb]: Shouldn't this be `Friday`?

File: app/views/pages/shecoded.html.erb [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        }
+    </style>
+  </head>
+  <body>
+    <div class="content">
+      <h1>Nevertheless, She Coded</h1>
+      <img src="<%= asset_path "SheCodedLogo.png" %>" alt="shecoded logo" />
+      <h2 class="mobile-only">👩🏿‍💻👩‍💻👩🏾‍💻</h2>
+      <div class="text">
+        <b>International Women's Day is next week and we want to celebrate the
```
Comment (Glennmen) [inline app/views/pages/shecoded.html.erb]: `next week` => `this week`

**Comment by benhalpern** [comment]
@Glennmen Thanks for the copy catches. We must have used last year's copy and forgot to change it?

**Pull request merged**